rest-server/src/main/java/io/github/nubesgen/configuration/NubesgenConfiguration.java

```diff
@@ -254,7 +275,7 @@ public boolean isRuntimePython() {
 
     @JsonIgnore
     public boolean isRuntimeDefault() {
```

Comment:
should not this be better as a method of `RuntimeType`. `RuntimeType::isDefault`?